code/modules/religion/religion_sects.dm

```diff
@@ -213,7 +216,107 @@
 	if(!offering.lit)
 		to_chat(user, "<span class='notice'>The candle needs to be lit to be offered!</span>")
 		return
-	to_chat(user, "<span class='notice'>Another candle for [GLOB.deity]'s collection</span>")
+	to_chat(user, "<span class='notice'>Another candle for [GLOB.deity]'s collection.</span>")
 	adjust_favor(20, user) //it's not a lot but hey there's a pacifist favor option at least
 	qdel(offering)
 	return TRUE
+
+/datum/religion_sect/greed
+	name = "Greedy God"
+	desc = "A very mercantile sect."
+	altar_icon_state = "convertaltar-yellow"
+	alignment = ALIGNMENT_EVIL //greed is not good wtf
+	rites_list = list(/datum/religion_rites/greed/vendatray, /datum/religion_rites/greed/custom_vending)
+	convert_opener = "\"Greed is good.\"<br>\
+	In the eyes of your mercantile diety, your wealth is your favor. Earn enough wealth to purchase some more business opportunities."
+	altar_icon_state = "convertaltar-red"
+
+/datum/religion_sect/honorbound
+	name = "Honorbound God"
+	desc = "A sect that does not engage in harm."
+	altar_icon_state = "convertaltar-white"
+	alignment = ALIGNMENT_GOOD
+	convert_opener = "\"A good, honourable crusade against evil is required.\"<br>\
+	Your diety requires fair fights from you. You may not attack the unready, the just, or the innocent.<br>\
+	You earn favor by getting others to join the crusade, and you may spend favor to announce a battle, bypassing some conditions to attack."
+	var/list/guilty = list() //list of guilty people
+
+/datum/religion_sect/honorbound/on_conversion(mob/living/new_convert)
+	RegisterSignal(new_convert, COMSIG_HUMAN_EARLY_UNARMED_ATTACK, .proc/unarmed_honor_check)
+	RegisterSignal(new_convert, COMSIG_ATOM_HULK_ATTACK, .proc/hulk_honor_check)
+	RegisterSignal(new_convert, COMSIG_MOB_ITEM_ATTACK, .proc/itemattack_honor_check)
+
+/datum/religion_sect/honorbound/proc/unarmed_honor_check(mob/living/carbon/human/honorbound_human, atom/target, proximity)
+	SIGNAL_HANDLER
+	if(!isliving(target))
+		return
+	var/mob/living/target_creature = target
+	if(!is_honorable(honorbound_human, target_creature))
+		return COMPONENT_CANCEL_ATTACK_CHAIN
+
+/datum/religion_sect/honorbound/proc/hulk_honor_check(atom/target, mob/living/carbon/human/honorbound_human)
+	SIGNAL_HANDLER
+	if(!isliving(target))
+		return
+	var/mob/living/target_creature = target
+	if(!is_honorable(honorbound_human, target_creature))
+		return COMPONENT_CANCEL_ATTACK_CHAIN
+
+/datum/religion_sect/honorbound/proc/itemattack_honor_check(atom/item, atom/target, mob/living/carbon/human/honorbound_human)
```

Comment:
make this check for the item's force? i dont think you'll be able to use healing stacks on people or anything

Reply:
good point thanks